index.rst

```diff
@@ -1,51 +1,387 @@
+
+.. math::
+   :label: eqn-basic_template
+
+    \parallel \overrightarrow{s_k}\!\!\parallel  = \sum_\alpha B_{k\alpha}  \overrightarrow{y_\alpha}
+ 
+
+Extension to variable seeing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While not implemented yet, there is a fairly clear path forward to extend the iterative solution from :eq:`eqn-iterative_sum` to the case where additional effects beyond DCR introduce changes to the PSF
+Variable seeing is the primary concern in this case, but in principle instrumental and other effects could be accounted for in this manner as well. 
+
+Now, we need to convolve the model with the measured PSF of the image :math:`Q^{(i)}`, and convolve the image with the fiducial PSF used for the model :math:`P`.
+This modifies :eq:`eqn-basic_sum` above:
+
+.. math::
+   :label: eqn-psf_sum
+
+   \sum_\alpha B_{i\alpha}  Q^{(i)} \overrightarrow{y_\alpha}  = P  \overrightarrow{s_i} 
+
+Now we can once again apply the reverse shift for one sub-band, and re-write :eq:`eqn-psf_sum` as:
+
+.. math::
+   :label: eqn-psf_iterative_sum
+
+    Q^{(i)}\overrightarrow{y_\gamma} = B_{\gamma i}^\star  P \overrightarrow{s_i} - B_{\gamma i}^\star  \sum_{\alpha  \neq \gamma} B_{i\alpha}  Q^{(i)} \overrightarrow{y_\alpha}  
+
+Unfortunately, we now have improved estimates for :math:`Q^{(i)}\overrightarrow{y_\gamma}` when what we really want is :math:`y_\gamma`.
+This problem is identical to the standard problem of image co-addition, however, so at this point we would hook in an existing algorithm for combining images with variable PSFs.
+
+Implementation
+--------------
+
+There are four main factors to consider when turning :eq:`eqn-iterative_sum` into an effective algorithm: 
+what initial solution to use as the starting point for iterations,
+what conditioning to apply to the new solution found in each iteration,
+how to detect and down-weight contaminated data,
+and how to determine when to exit the loop.
+These factors will each be described in a subsection below.
+
+
+Finding the initial solution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assuming we have no prior spectral information, the best initial guess is that all sub-bands have the same flux in all pixels.
+If all model planes are equal at the start, a good guess for the flux distribution within a sub-band is the standard co-add of the input images, divided by the number of model planes being used (since those will be summed).
+A proper inverse-variance weighting of the input images as part of the coaddition will help make the best estimate, and if there are many input images we could restrict the coaddition to use only those observed near zenith (with negligible DCR).
+An advantage of selecting the simple coadd as the starting point, is that the solution should immediately converge if the input data exhibits no actual DCR effects, such as *i*-band or zenith observations.
```

Comment:
It's probably worth saying "redder bands (i and redder for LSST)", rather than specifically i-band.